contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1519,11 +1558,34 @@ has_unique_nullable_constraint(ColumnDef *column)
 	return is_unique & !is_notnull;
 }
 
+static bool
+has_nullable_constraint(ColumnDef *column)
+{
+	ListCell   *clist;
+	bool		is_notnull = false;
+
+	foreach(clist, column->constraints)
+	{
+		Constraint *constraint = lfirst_node(Constraint, clist);
+
+		switch (constraint->contype)
+		{
+			case CONSTR_NOTNULL:
+				is_notnull = true;
+				break;
+			default:
+				break;
+		}
+	}
+
+	return !is_notnull;
+}
+
 static bool
 is_nullable_constraint(Constraint *cst, Oid rel_oid)
```

Comment:
You've completely changed the definition of the function. Is that expected? Also, could you please add a top-level comment what the function is trying to do?

Reply:
Yes it was expected, as is_nullable_constraint was initially returning true only if all the columns in the constraint is null else false. But expected behaviour should be return true if atleast one of the column is null else false(if all the columns are not null)

Reply:
Added a top-level comment